nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/c2/C2NifiClientService.java

```diff
@@ -162,4 +173,26 @@ private Map<String, FlowQueueStatus> getQueueStatus() {
 
         return processGroupStatus;
     }
+
+    private boolean updateFlowContent(ByteBuffer updateContent) {
```

Comment:
For now this is the file write + restart approach. As it is not final and we will go with flowController / flowService based reload I tried to keep it as simple as possible.